$NetBSD: patch-al,v 1.1 2008/03/18 11:48:25 dmcmahill Exp $

The automake manual clearly specifies that if we mix fortran sources
in with c or c++ sources then we need to use the AC_F77_LIBRARY_LDFLAGS
autoconf macro and then add $(FLIBS) to either LDADD for a program or
LIBADD for a library.

--- bem/Makefile.am.orig	2004-04-22 20:07:12.000000000 +0000
+++ bem/Makefile.am	2008-03-16 03:44:18.000000000 +0000
@@ -27,5 +27,5 @@
 #  supply command line switches --add-missing --copy.
 #----------------------------------------------------------------
-AUTOMAKE_OPTIONS = foreign subdir-objects
+AUTOMAKE_OPTIONS = foreign
 
 #---------------------------------------------------------------
@@ -72,7 +72,9 @@
 bem_SOURCES = $(cpp_SOURCES) $(fortran_SOURCES) \
 		src/nmmtl_parse_xsctn.cpp
+bem_LDADD = $(FLIBS)
 
 nmmtl_SOURCES = $(cpp_SOURCES) $(fortran_SOURCES) \
 		src/nmmtl_parse_graphic.cpp
+nmmtl_LDADD = $(FLIBS)
 
 
